llvm/lib/CodeGen/MachineOperand.cpp

```diff
@@ -1240,7 +1240,11 @@ void MachineMemOperand::print(raw_ostream &OS, ModuleSlotTracker &MST,
        << "unknown-address";
   }
   MachineOperand::printOperandOffset(OS, getOffset());
-  if (getSize() > 0 && getAlign() != getSize())
+  uint64_t MinSize = MemoryType.isValid()
+                         ? getType().getSizeInBytes().getKnownMinValue()
+                         : ~UINT64_C(0);
+  // TODO: getSize should return TypeSize
```

Comment:
Should we move the TODO to the declaration of getSize, since it isn't called here anymore?

Reply:
sounds more reasonable!